js/exporter/jspdf/v3/rows_generator.js

```diff
@@ -62,16 +62,16 @@ function generateRowsInfo(dataProvider, dataGrid, headerBackgroundColor) {
     return result;
 }
 
-function generateRowCells({ dataProvider, rowIndex, wordWrapEnabled, colCount, rowType, backgroundColor }) {
+function generateRowCells({ dataProvider, rowIndex, wordWrapEnabled, columns, colCount, rowType, backgroundColor }) {
```

Comment:
if we pass whole columns, then colCount argument is redundant.

Reply:
Ok